packages/core/src/builder.class.ts

```diff
@@ -1663,9 +1663,9 @@ export class Builder {
       this.authToken = authToken;
     }
     if (isBrowser) {
-      this.bindMessageListeners();
-
       if (Builder.isEditing) {
+        this.bindMessageListeners();
```

Comment:
moving this call inside of `Builder.isEditing` is the fundamental fix in this PR: before that, we were always listening to `builder.*` messages like `builder.evaluate`.